CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
Ah I actually only get a warning, not a build error. So everything is cool, I'll just ignore the warnings on my github :)